### DISCUSSION TITLE
Folder specific deplyoments

### DISCUSSION THREAD

**Discussion by prince-bansal** [vercel/community#6646] [Help] [answered]
### Summary

Hi there, I have a mono repo project with two projects, one for the user and one for the seller. Is there a way on Vercel where we can deploy the user folder to a different domain and the seller folder to a different domain?

Example -> 

```
/user     -> app.user.com
/seller   -> app.seller.com
```

### Steps to Reproduce

Have a monorepo setup, create two nested folders with both having a next application. Need to deploy those folders to different domains.

**Comment by amyegan** [comment, marked as answer]
Yes, you can! We have some documentation about how to use monorepos with Vercel at [vercel.com/docs/monorepos](https://vercel.com/docs/monorepos)
